ui/components/home-office.jsx

```diff
@@ -24,6 +24,7 @@ class HomeOffice extends React.Component {
         <body>
 
           <header role="banner" className={this.props.propositionHeader ? 'with-proposition' : ''}>
+            <a href={this.props.skipToContentTarget} className="govuk-skip-link">{this.props.skipToContent}</a>
```

Comment:
This should probably have a default value of a thing that reliably exists.

Reply:
see https://github.com/UKHomeOffice/asl-service/blob/master/ui/components/home-office.jsx#L131-L132

Reply:
Links to https://github.com/UKHomeOffice/asl-service/blob/master/ui/views/base.jsx#L73

Reply:
Oh, so we theoretically already had this, it was just broken?

Reply:
Parts of it were already there, I assume it was either added in prep for this, or the link was removed at some point, but I couldn't find any evidence when or where.